src/platform/site-wide/va-footer/containers/Main.jsx

```diff
@@ -62,7 +54,7 @@ export class Main extends React.Component {
             <li className="usa-accordion-content" id="veteran-crisis" aria-hidden="true">
               <ul className="va-footer-links">
                 <li>
-                  <a role="button" tabIndex="0" onClick={this.openModal} onKeyPress={this.handleKeyPress} >Veterans Crisis Line</a>
+                  <button className="va-button-link va-overlay-trigger" data-show="#modal-crisisline">Veterans Crisis Line</button>
```

Comment:
Our usual practice is to make these sorts of things buttons, then style them as links.